website/static/js/projectorganizer.js

```diff
@@ -1254,6 +1453,19 @@ var tbOptions = {
     lazyLoadOnLoad : expandStateLoad,
     resolveRefreshIcon : function() {
         return m('i.fa.fa-refresh.fa-spin');
+    },
+    headerTemplate : _poToolbar,
+    // Not treebeard options, specific to Fangorn
+    iconState : {
+        mode : 'bar',
+        generalIcons : {
+            search : { on : true, template : searchButton }         
+        },
+        rowIcons : [{}]
+        },
+    defineToolbar : _poDefineToolbar,
+    onselectrow : function(row) {
+        console.log(row);
```

Comment:
That log is very useful while PR is in WIP mode, I'll take it out at the very end.